model.go

```diff
@@ -154,6 +154,8 @@ type Model interface {
 	PasswordHash() string
 
 	AddBlockDevice(string, BlockDeviceArgs) error
+
+	SetEnvironVersion(environVersion int)
```

Comment:
Can you please add some documentation to this? I understand the standard set previously is to be very wishy washy with this but we are trying to do better and because this is new code we should make the effort.

I would suggest documenting the pre and post side affects of this func.